### PR TITLE
rerun add_docs.py

### PR DESCRIPTION
**Pull request by Qalthos**
Depends-On: https://github.com/ansible-collections/ansible.netcommon/pull/54
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

everything

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

**Comment by Qalthos** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -4,24 +4,35 @@
 [![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/builds?project=ansible-collections%2Fansible.netcommon) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/ansible.netcommon)](https://codecov.io/gh/ansible-collections/ansible.netcommon)-->
 
 The Ansible ``ansible.netcommon`` collection includes common content to help automate the management of network, security, and cloud devices.
-This includes  connection plugins, such as ``network_cli``, ``httpapi``, and ``netconf``.
+
+<!--start requires_ansible-->
+## Ansible version compatibility
+
+This collection has been tested against following Ansible versions: **>=2.9,<2.11**.
```
Comment (GomathiselviS) [inline README.md:11]: add_docs should be rerun. The latest update will have **>=2.9.10,<2.11**.

File: docs/ansible.netcommon.default_netconf.rst [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
 .. _ansible.netcommon.default_netconf:
```
Comment (GomathiselviS) [inline docs/ansible.netcommon.default_netconf.rst:1]: latest update to add_docs will remove the 'hint' part from the document.

Comment (Qalthos) [reply docs/ansible.netcommon.default_netconf.rst:1]: Hmmm, maybe it isn't triggering on non-modules

File: README.md (hunk 1 above)
Comment (Qalthos) [reply README.md]: Ah, I needed to rerun runtime again first.

File: docs/ansible.netcommon.default_netconf.rst (hunk 2 above)
Comment (GomathiselviS) [reply docs/ansible.netcommon.default_netconf.rst:1]: yes , you are right.

**Pull request merged**